### PR TITLE
Add cypress tests for company interest create, delete and edit

### PR DESCRIPTION
**Pull request by mrfarstad**

### CODE REVIEW

**Comment by odinuge** [comment]
:tada:

**Comment by orhanhenrik** [comment]
We should probably also test this page without being signed in, since most users of this page will not have an abakus user

**Comment by mrfarstad** [comment]
> We should probably also test this page without being signed in, since most users of this page will not have an abakus user

Yep, we definitely should! :)

However, the tests seem to be non-deterministic, so I think I will close this PR

File: app/routes/companyInterest/components/CompanyInterestPage.js [hunk 1]
```diff
@@ -363,5 +363,6 @@ const validate = createValidator({
 export default reduxForm({
   form: 'CompanyInterestForm',
   validate,
-  enableReinitialize: true
+  enableReinitialize: true,
+  keepDirtyOnReinitialize: true
```
Comment (odinuge) [inline app/routes/companyInterest/components/CompanyInterestPage.js]: Dirty stuff :fire:

**Pull request merged**